src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-tabs/smb-tabs.component.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SmbTabsComponent } from './smb-tabs.component';
+import { By } from '@angular/platform-browser';
+
+describe('SmbTabsComponent', () => {
+  let component: SmbTabsComponent;
+  let fixture: ComponentFixture<SmbTabsComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [SmbTabsComponent]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(SmbTabsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should display the heading text in the tab', () => {
+    const tabs = fixture.debugElement.queryAll(By.css('cds-tab'));
+    expect(tabs.length).toBe(3);
+    expect(tabs[0].attributes['heading']).toBe('Clusters');
+    expect(tabs[1].attributes['heading']).toBe('Active Directory');
+    expect(tabs[2].attributes['heading']).toBe('Standalone');
+  });
+
+  // If the cache is actvie it will try to fetch for every tab at once
+  // the smb module does not like that when trying to request the smb db
```

Comment:
This comment seems confusing, can we improve ?

Reply:
Changed it from:
```
// If the cache is actvie it will try to fetch for every tab at once
 // the smb module does not like that when trying to request the smb db
```

To now:
```
 // If the tabs cacheActive is set to true data for all tabs will be fetched at once,
 // smb mgr module might hit recursive error when doing multiple request to the db
  ```
  
  Hopefully that is more clear, and about this issue, I'll let John now.